vendor/assets/javascripts/brainstem/brainstem-model.coffee

```diff
@@ -95,6 +105,16 @@ class window.Brainstem.Model extends Backbone.Model
     else
       super(field)
 
+  set: (key, val, options) ->
+    super
+
+    _.extend(@unsavedAttributes || {}, @changed)
```

Comment:
When the model itself invokes `set` its constructor, however we need to call its constructor first so we can make sure the constructor does not set `unsavedAttributes` to attributes passed to the constructor.